src/providers/authentication/__tests__/authentication.spec.ts

```diff
@@ -53,6 +67,20 @@ describe('Authentication', () => {
       expect(authenticationProvider.isAuthenticated()).toEqual(true);
     });
 
+    it('should login without authenticating in unauthenticated mode', async () => {
+      spyOn(authenticationProvider, 'isInUnAuthenticatedMode').and.returnValue(true);
+      spyOn(authenticationProvider, 'aquireTokenSilently').and.callThrough();
+      await authenticationProvider.login();
+      expect(authenticationProvider.aquireTokenSilently).toHaveBeenCalledTimes(0);
+    });
+
+    it('should login without authenticating in unauthenticated mode', async () => {
```

Comment:
think the it statement should say 'should login **with** authenticating in unauthenticated mode'